tools/topology/m4/eq_fir.m4

```diff
@@ -28,9 +28,9 @@ define(`W_EQ_FIR',
 `	tuples "'N_EQ_FIR($1)`_tuples_str"'
 `}'
 `SectionVendorTuples."'N_EQ_FIR($1)`_tuples_str_type" {'
-`	tokens "sof_effect_tokens"'
+`	tokens "sof_process_tokens"'
```

Comment:
Already covered by #1079.

Reply:
yes, as it has dependency, I apply this on top of #1079 applied, will update it once #1079 merged.